packages/scan-request-sender/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "description": "This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.",
     "scripts": {
-        "build": "webpack --config ./webpack.config.js --config-name scanner --colors --progress",
```

Comment:
config-name parameter is ignored, if the webpack config exports an object instead of array of objects